modal/config.py

```diff
@@ -242,6 +242,7 @@ class _Setting(typing.NamedTuple):
     "cuda_checkpoint_path": _Setting("/__modal/.bin/cuda-checkpoint"),  # Used for snapshotting GPU memory.
     "function_schemas": _Setting(False, transform=_to_boolean),
     "build_validation": _Setting("error", transform=_check_value(["error", "warn", "ignore"])),
+    "internal_experimental_input_plane_url": _Setting(None),  # For internal/experimental use
```

Comment:
We have a few other "internal" config settings above that we don't distinguish based on the name. We should probably clean that up somehow but I think better to be consistent with existing conventions. (The config options are public in some sense, but the documentation is opt-in).

Reply:
makes sense, I renamed to `input_plane_url`. thanks!